configuration.adoc

```diff
@@ -1803,6 +1810,101 @@ Example, "userSMIMECertificate, UserCertificate"
 
 To set up another external LDAP server, click *Add Configuration*.
 
+=== Setting up server based S/MIME feature
+
+==== Prerequisites
+
+Same as the previous S/MIME version except that the Java is not required on client machine
+
+==== S/MIME License
+
+Same as the previous S/MIME version
+
+==== Enable S/MIME Feature
+
+Same as the previous S/MIME version
+
+==== Importing S/MIME Certificates
+
+Same as the previous S/MIME version except that the recipients' public-key certificate no longer need to be stored into Local OS or browser keystore.
+The certificate can be published to all other places mentioned in previous S/MIME version.
+
+==== List of LDAP attributes introduced in new S\MIME feature
+
+. zimbraSmimeOCSPEnabled
+
+* Used by server at the time of validating the user as well as public certificates
+* If TRUE, the revocation check will be performed during certificate validation
+* If FALSE, the revocation check will not be performed during certificate validation
+
+. zimbraSmimePublicCertificateExtensions
+
+* The supported public certificate file extensions separated by comma
+* Contains the list of supported formats for userCertificate LDAP attribute
+* Default values: cer,crt,der,spc,p7b,p7r,sst,sto,pem
+* Zimbra web client retrieves from server the supported file formats/extensions for public certificate upload
+
+. zimbraSmimeUserCertificateExtensions
+
+* The supported public certificate file extensions separated by comma.
+* Contains the list of supported formats for userSmimeCertificate LDAP attribute
+* Default values: p12,pfx
+* Zimbra web client retrieves from server the supported file formats/extensions for user certificate upload
+
+==== Process for Adding the CA certificate to the mailbox truststore for S/MIME
+
+S/MIME uses the mailbox trust store path and its password defined in localconfig.xml
+
+The key names are:
+
+* mailboxd_truststore
+* mailboxd_truststore_password
+
+If the key is not defined in localconfig.xml, by default the values of mailboxd_truststore and mailboxd_truststore_password are:
+
+* <zimbra_java_home>/jre/lib/security/cacerts
+* changeit
+
+CA certificate can be imported to mailbox trust store by executing following command:
+
+keytool -import -alias <alias> -keystore <mailboxd_truststore path> -trustcacerts -file <CA_Cert>
+
+== Admin SIEVE Filter
+
+* The admin SIEVE feature enables the system administrator to define one SIEVE rule executed before and one SIEVE rule executed after the end user-defined filter rule.
+* The system-admin-defined SIEVE rule will be combined with the end-user-defined SEIVE rule, and then combined one big SIEVE rule will be applied to both incoming and outgoing messages. But, the scope of the variables is local within each script (zimbraMailAdminSieveScriptBefore, zimbraMailSieveScript, zimbraMailAdminSieveScriptAfter) OR (zimbraMailAdminOutgoingSieveScriptBefore, zimbraMailOutgoingSieveScript, zimbraMailAdminOutgoingSieveScriptAfter)
```

Comment:
Suggest for the first sentence:
The system-admin-defined SIEVE rule will be combined with the end-user-defined SEIVE rule into a single rule. This rule will be applied to both incoming and outgoing messages